docs/docs/references/configuration/config-file.md

```diff
@@ -327,6 +448,63 @@ kubernetes:
   # Same as '--namespace'
   # Default is empty
   namespace:
+
+  # Same as '--kubeconfig'
+  # Default is empty
+  kubeconfig: ~/.kube/config2
+
+  # Same as '--components'
+  # Default is 'workload,infra'
+  components: 
+    - workload
+    - infra
+
+  # Same as '--tolerations'
+  # Default is empty
+  tolerations:
+    - key1=value1:NoExecute
+    - key2=value2:NoSchedule
+
+  # Same as '--qps'
+  # Default is 5.0
+  qps: 5.0
+
+  # Same as '--burst'
+  # Default is 10
+  burst: 10
+
+  exclude:
+    # Same as '--exclude-owned'
+    # Default is false
+    owned: true
+
+    # Same as '--exclude-nodes'
+    # Default is empty
+    nodes:
+      - kubernetes.io/arch:arm64
+      - team:dev
+
+  # Same as '--all-namespaces'
+  # Default is false
+  all:
+    namespaces: false
+
+  # Same as '--k8s-version'
+  # Default is empty
+  k8s:
+    version: 1.21.0
+    
+  # Same as '--node-collector-imageref'
+  # Default is 'ghcr.io/aquasecurity/node-collector:0.0.9'
+  node:
+    collector:
+      imageref: ghcr.io/aquasecurity/node-collector:0.0.9
+
+# Same as '--node-collector-namespace'
+# Default is 'trivy-temp'
+node:
+  collector:
+    namespace: ~/.kube/config2
```

Comment:
@chen-keinan can you take a look? Looks like we don't need nested structs for these flags. Perhaps we can use `-` instead of `.`.

e.g. for `--node-collector-imageref`:
 https://github.com/aquasecurity/trivy/blob/fae710db8fcb6560df7b69355374760c0f2a84c9/pkg/flag/kubernetes_flags.go#L77-L82

we can use `kubernetes.node-collector-imageref` and get :
```yaml
kubernetes 
 node-collector-imageref: ghcr.io/aquasecurity/node-collector:0.0.9
```

Reply:
@DmitriyLewen sound good. you can update this flag as well.

Reply:
@chen-keinan take a looks, please - https://github.com/aquasecurity/trivy/pull/6547/commits/15e6fcaa7e130b660b9d6d35b1270654df00d726